include/raft_private.h

```diff
@@ -91,11 +91,6 @@ int raft_send_appendentries(raft_server_t* me, raft_node_t* node);
 
 int raft_send_appendentries_all(raft_server_t* me_);
 
-/**
- * Apply entry at lastApplied + 1. Entry becomes 'committed'.
- * @return 1 if entry committed, 0 otherwise */
-int raft_apply_entry(raft_server_t* me_);
-
```

Comment:
See comment above. While it is okay to commit one entry at a time, we really don't want to be applying entries discretely -- 
1. because it is slow 
2. because we would then have to handle membership changes quite differently instead of how it is currently done I saw a lot of segfaults and other errors when testing membership changes especially repeatedly adding and removing the same replica ranks, and during installsnapshot / appendentries processing. I figured the best way to handle it in the current scheme of things is to simply track membership changes until all logs are applied and then update the final state after the last committed log entry is applied.